validation-test/SIL/verify_all_overlays.sil

```diff
@@ -1,3 +1,3 @@
-// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | FileCheck %s; %target-sil-opt -enable-sil-verify-all $x > /dev/null; done
+// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | FileCheck %s; %target-sil-opt -sdk %sdk -enable-sil-verify-all $x > /dev/null; done
```

Comment:
The "target" in `target-sil-opt` is supposed to imply that the SDK is included; want to just change the substitution?